### DISCUSSION TITLE
Open an url with a default page zoom size?

### DISCUSSION THREAD

**Discussion by Twix53791** [qutebrowser/qutebrowser#7164] [Q&A] [answered]
Hi, is there a better way than the code below to open, from outside, from a script, an url and setting the page zoom in the same time? ` qutebrowser ":open url" ":zoom 125" does not work, it seems the page needs first to be loaded before the applied zoom takes effect?

```
qutebrowser ":open $url"
sleep .3; qutebrowser ":zoom -q 125"
```
Thanks

**Comment by toofar** [comment, marked as answer]
You can use the qutebrowser :later command for the delay instead of sleep. But I don't think there is currently a way to control it better than that. (So something like `:open -t example.com ;; later 3 zoom -q 125`)
